apis/filesystem/src/main/java/org/jclouds/filesystem/strategy/internal/FilesystemStorageStrategyImpl.java

```diff
@@ -745,28 +743,26 @@ protected String buildPathStartingFromBaseDir(String... pathTokens) {
    }
 
    /**
-    * Substitutes all the file separator occurrences in the path with a file separator for the
-    * current operative system
+    * Convert path to the current OS filesystem standard
     *
-    * @param pathToBeNormalized
+    * @param path
     * @return
     */
-   private static String normalize(String pathToBeNormalized) {
-      if (null != pathToBeNormalized && pathToBeNormalized.contains(BACK_SLASH)) {
-         if (!BACK_SLASH.equals(File.separator)) {
-            return pathToBeNormalized.replace(BACK_SLASH, File.separator);
-         }
+   private static String normalize(String path) {
+      if (null != path) {
```

Comment:
This will also normalize the path on a Linux system, so I would rather leave it as is